registration.html

```diff
@@ -0,0 +1,175 @@
+	<!-- <script defer src="assets/fontawesome-free-5.15.2-web/js/all.js"></script> -->
+	</head>
+	<body class="is-preload">
+
+		<!-- Wrapper -->
+			<div id="wrapper">
+
+				<!-- Main -->
+					<div id="main">
+						<div class="inner">
+
+							<!-- Header -->
+ 								<header id="header">
+									<a href="index.html" class="logo"><img src='images/test.png'></img>MPI-SHH Summer School</a>
+									<!-- <ul class="icons">
+										<li><a href="#" class="icon brands fa-twitter"><span class="label">Twitter</span></a></li>
+									</ul> -->
+								</header>
+                                
+                                <section id='introduction'>
+                                    <header class="major">
+										<h2>About MPI-SHH Summer School</h2>
+									</header>
+                                    <article>
+                                        <p>Our teachers are mainly doctoral students and will teach you theoretical knowledge about their field, methods they use, and the soft skills they developed. The school will be held online from <b>August 16 to 27, 2021</b>. The first week will consist of lectures introducing the main aspects of archaeology, archaeogenetics as well as linguistic and cultural evolution. 
+										In the second week, the focus will be on case studies illustrating interdisciplinary projects that combine all three fields in a geographical area (e.g. Southeast Asia). Workshops on scientific methods (e.g., ZooMS, Bayesian phylogenetics) and soft-skill courses (e.g., LaTeX, graphics, research ethics) will be scheduled over the two weeks. 
+										Each day will consist of 3-4 courses in the time between 9 am to 5 pm with breaks in between. And there will be no parallel sessions so that you are able to attend all courses!</p>
+										<p>Besides, you will hear three invited keynotes from distinguished researchers and listen to a round table discussion between three senior scientists from our institute. The program is rounded off by social events in the evening where you can chat with fellow students, teachers, and the organizing team.  
+										To receive course credit for the summer school, you will need to prepare a poster on a topic of interest and present it at our poster session on the last day of school.</p>
+										
+										<p>If you are interested in knowing more about our institute, press the button below to view our institute's website.</p>
+										<a href="https://www.shh.mpg.de/en" class="button big"> more</a></br>
+                                        
+                                        </article>
+                                </section>
+
+                                <section id='qualification'>
+                                    <header class="major">
+										<h2>Who can join?</h2>
+									</header>
+                                        <article>
+                                            <p>The summer school is intended for students:
```

Comment:
I'd suggest we keep it simple and say: 

The summer school is intended for Master's and doctoral students in archaeology, archaeogenetics, biology, informatics, linguistics, or related disciplines. We also welcome undergraduate students with the aforementioned backgrounds. Each applicant should have an interest in interdisciplinary studies and want to discover new disciplines to expand their knowledge.